src/shared/ContentsTable/TableEntries/BranchEntries/BranchFileEntry.jsx

```diff
@@ -12,10 +12,14 @@ function BranchFileEntry({
   name,
   urlPath,
   displayType,
+  filters,
 }) {
+  const flags = filters?.flags?.length > 0 ? filters?.flags : []
```

Comment:
nitpick: `filter.flags` is fine after the `?` without the optional chaining as we know it exists